KomaMRICore/src/datatypes/Magnetization.jl

```diff
@@ -1,3 +1,5 @@
+abstract type SpinStateRepresentation{T<:Real} end #get all available types by using subtypes(KomaMRI.SpinStateRepresentation)
```

Comment:
Why did you move this? revert. `abstract type SpinStateRepresentation{T<:Real}` should be independant of `Mag`. The file for `Mag` is meant as a template of everything you need to define for a `SpinStateRepresentation`, so a new user should copy it and start with that. If the definition of the abstract type `SpinStateRepresentation` is in the file the user will redefine it.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/738f91116f5f71e5ae8ec1bda6bbc5d78a83a10d